typewise_alert.test.py

```diff
@@ -4,8 +4,28 @@
 
 class TypewiseTest(unittest.TestCase):
   def test_infers_breach_as_per_limits(self):
+    self.assertTrue(typewise_alert.infer_breach(0,0, 35) == 'NORMAL')
+    self.assertTrue(typewise_alert.infer_breach(-1,0, 35) == 'TOO_LOW')
+    self.assertTrue(typewise_alert.infer_breach(30, 0,35) == 'NORMAL')
+    self.assertTrue(typewise_alert.infer_breach(35,0,35) == 'NORMAL')
+    self.assertTrue(typewise_alert.infer_breach(37,0,35) == 'TOO_HIGH')
+    self.assertFalse(typewise_alert.check_and_alert('TO_EMAIL', 'MED_ACTIVE_COOLING', -1) == 'TOO_LOW')
```

Comment:
its better to have negative tests. But we need to ensure the positives first. In this assert, the return value should be normal , so assert false seems fine in first glance. But what if accidentally the production code return TOO_HIGH ?
Hence it is better to assert the behaviour first and then proceed to negative tests.